marketplace/deployer_util/expand_config.py

```diff
@@ -163,6 +165,17 @@ def validate_value_types(values, schema):
               k, prop.type, v))
 
 
+def maybe_derive_deployer_image(schema, values_dict):
```

Comment:
Is this `maybe_...` a convention somewhere else?

Could you use this instead:
`derive_deployer_image_if_applicable`

Reply:
Resolved offline.